types/gapi/index.d.ts

```diff
@@ -35,10 +35,20 @@ interface GoogleApiOAuth2TokenObject {
  */
 declare namespace gapi {
 
+    type LoadCallback = () => void | Function;
```

Comment:
I know this is kind of a weird declaration, `() => void` is technically correct. However many consumers use something like:

``` TypeScript
new Promise((resolve) => {
  gapi.load('thing', resolve)
})
```

and `resolve` does not have the type of `() => void`, so I added `Function` as a blanket catchall.